src/elements/content-explorer/PreviewDialog.scss

```diff
@@ -0,0 +1,16 @@
+[class^='bp_modal_module_content'].bce-PreviewDialog {
```

Comment:
we can't rely on class name or specificity as a long term solution

Reply:
do you have a suggestion in mind about this? could try to target the order of elements but that is unreliable too, VRTs should fail in either case though